README.md

```diff
@@ -1,75 +1,53 @@
 # Phoxygen Website
 
-This is the official Phoxygen website.
+This is the official Phoxygen website, to be published at http://www.phoxygen.com/.
 
-We started from the Jekyll implementation of the [Stylish
-Portfolio](http://startbootstrap.com/template-overviews/stylish-portfolio/)
-template by [Start Bootstrap](http://startbootstrap.com/).
+Template based on [Stylish
+Portfolio](http://startbootstrap.com/template-overviews/stylish-portfolio/).
 
-See the site in action at http://www.phoxygen.com/
+## Developer Workflow
 
-## Dev Workflow
-
-The website can be accessed on http://localhost:4000/
-
-You can either install the dependencies manually or use the Docker recipe.
+Jekyll is required to generate this website.
+It can be either installed manually or run from a Docker container.
 
 ### Manual Install
 
-Prerequisite:
-
-- Ruby 2.1.0 or later
-- bundler
+Prerequisite: Ruby 2.1.0 or later
 
 ```bash
 gem install bundler
 bundle install
-# jekyll serve
-bundle exec jekyll serve --watch
+bundle exec jekyll serve
 ```
 
-### With Docker
-
-You can let docker deal with the ruby gems.
-
-**/!\ Make sure your Docker client points to a local Docker daemon.**
-These steps are not suitable for a remote docker instance.
+Note: `jekyll serve` should be equivalent to the last line but might raise
+a dependency issue (e.g. with kramdown).
 
-The easiest way to do so is to check what’s inside the DOCKER_HOST environment variable.
-
-#### The Easy Way: docker-compose
+### With Docker
 
-At the root of the repository:
+Pull the official [Jekyll container](https://github.com/jekyll/docker#running)
+— it even has a specific tag for github-pages:
 
 ```bash
-docker-compose up
+docker pull jekyll/jekyll:pages
 ```
 
-builds a docker image if not already done and starts the server on port 4000.
-This is strictly equivalent to using `jekyll serve` (i.e. it listens to your local
-changes, etc.).
-
-Edit `docker-compose.yml` to change settings if the defaults do not suit your needs.
-
-#### Without docker-compose
-
-These docker commands are strictly equivalent to the `docker-compose up` above.
+On Native Docker:
 
 ```bash
-# run this once
-docker build -t <my-tag>
-
-# run this every time you want to start the server
-docker run -i -p 4000:4000 -v <absolute-path-to-project>:/home/user/src \
-<my-tag> serve -H 0.0.0.0
+docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll \
+  -it -p 127.0.0.1:4000:4000 \
+  jekyll/jekyll:pages bundle exec jekyll serve
 ```
 
-**Pro tip:**
-
-The entry point of the Dockerfile is `jekyll`. So you can alias `docker run -i -p
-4000:4000 -v <absolute-path-to-project>:/src grahamc/jekyll:latest` to
-`my-jekyll-alias` and use it just as you would use a local jekyll install:
+On Docker-Machine:
```

Comment:
So as I expected, I have the same error than when my docker client points to a remote machine. Actually docker-machine is just a wrapper around a wm (or a remote machine). So from docker's POV, this is really the "remote setup" we've talked about.

I got it partially working though: I rely on the fact that:
- you can mount your host home inside the vm
- the `--volume` arg actually works in a remote setup, but should actually be a folder on the machine where the docker daemon run (*not* the client). I should have started by that in our previous discussion.

So assuming you use the Virtualbox driver for docker-machine, once your vm has started:

```bash
# ssh into the machine
docker-machine ssh <machine-name> 
# create your mount point
mkdir hosthome
# mount your home folder, the share is already configured by docker-machine
# (you can modify it in virtualbox interface)
sudo mount  -t vboxsf -o uid=1000,gid=1000 hosthome hosthome
```

In your host (your machine running the docker client, typically also the host in the vm POV):
```bash
docker run --rm --label=jekyll --volume=/home/docker/hosthome/<path>:/srv/jekyll \
  -it -p $(docker-machine ip `docker-machine active`):4000:4000 \
  jekyll/jekyll:pages bundle exec jekyll serve
```
where <path> is the absolute path to your project without the `/home` part. So in my case the full volume arg is `--volume=/home/docker/hosthome/augustin/workspace/devwww:/srv/jekyll`

The server is starting, but I loose any `--watch` capabilities... After some digging, I found out that [inotify is not supported in vboxfs share](https://www.virtualbox.org/ticket/10660?cversion=0&cnum_hist=1)... So close, yet so far.

However, we can still keep these augmented instructions for Loic. Your call :-)